src/plugins/integrations/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './cocoapods';
-export * from './packageJson';
```

Comment:
did you remove the existing packageJson? not seeing it in the negative diff

Reply:
I kinda renamed it to something similar to what is in main

Reply:
@jwahjudi right but it looks like the old file isn't removed? `packageJson.ts` ?

Reply:
Ah i see, must have gotten added back from the merge conflict